samples/llama_index_quick_start.ipynb

```diff
@@ -0,0 +1,1014 @@
+    "    password=password,\n",
+    ")\n",
+    "\n",
+    "engine.init_vector_store_table(\n",
+    "    sample_vector_table_name,\n",
+    "    vector_size=768,\n",
+    "    metadata_columns=[\n",
+    "        Column(\"show_id\", \"VARCHAR\", nullable=True),\n",
+    "        Column(\"type\", \"VARCHAR\", nullable=True),\n",
+    "        Column(\"country\", \"VARCHAR\", nullable=True),\n",
+    "        Column(\"date_added\", \"VARCHAR\", nullable=True),\n",
+    "        Column(\"release_year\", \"INTEGER\", nullable=True),\n",
+    "        Column(\"rating\", \"VARCHAR\", nullable=True),\n",
+    "        Column(\"duration\", \"VARCHAR\", nullable=True),\n",
+    "        Column(\"listed_in\", \"VARCHAR\", nullable=True),\n",
+    "    ],\n",
+    "    overwrite_existing=True,  # Enabling this will delete and recreate the table if exists.\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "KG6rwEuJLNIo"
+   },
+   "source": [
+    "### Create the vector store instance\n",
+    "\n",
+    "Next, you will create a `AlloyDBVectorStore` object that connects to the new AlloyDB database table to store the data from the documents."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Wo4-7EYCIFF9"
+   },
+   "outputs": [],
+   "source": [
+    "from llama_index_alloydb_pg import AlloyDBVectorStore, AlloyDBEngine\n",
+    "\n",
+    "engine = AlloyDBEngine.from_instance(\n",
+    "    project_id=project_id,\n",
+    "    instance=instance_name,\n",
+    "    region=region,\n",
+    "    cluster=cluster_name,\n",
+    "    database=database_name,\n",
+    "    user=\"postgres\",\n",
+    "    password=password,\n",
+    ")\n",
+    "\n",
+    "vector_store = AlloyDBVectorStore.create_sync(\n",
+    "    engine=engine,\n",
+    "    table_name=sample_vector_table_name,\n",
+    "    metadata_columns=[\n",
+    "        \"show_id\",\n",
+    "        \"type\",\n",
+    "        \"country\",\n",
+    "        \"date_added\",\n",
+    "        \"release_year\",\n",
+    "        \"rating\",\n",
+    "        \"duration\",\n",
+    "        \"listed_in\",\n",
+    "    ],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "fr1rP6KQ-8ag"
+   },
+   "source": [
+    "#### Create a LlamaIndex `Index`\n",
+    "\n",
+    "An `Index` allows us to quickly retrieve relevant context for a user query. They are used to build `QueryEngines` and `ChatEngines` over which a user can get answers to their queries.\n",
+    "For a list of indexes that can be built in LlamaIndex, see [Index Guide](https://docs.llamaindex.ai/en/stable/module_guides/indexing/index_guide/).\n",
+    "\n",
+    "A `VectorStoreIndex`, can be built using the `AlloyDBVectorStore`. You can also use the `AlloyDBDocumentStore` and `AlloyDBIndexStore` to persist documents and index metadata. These modules can be used to build other `Indexes`.\n",
```

Comment:
We should set up the documentstore in this tutorial as well

Reply:
PTAL